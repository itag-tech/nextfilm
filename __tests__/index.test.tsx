import { render } from "@testing-library/react"
import { describe, expect, it } from "vitest"
import HomePage from "../pages/index"
import App from "../pages/_app"

describe("Home page", () => {
  it("should render as usual", () => {
    // when ----------------
    // @ts-ignore - FIXME: fix this typescript error
    const { container } = render(<App Component={HomePage} pageProps={null}></App>)

    // then ----------------
    expect(container).toMatchSnapshot()
  })
})