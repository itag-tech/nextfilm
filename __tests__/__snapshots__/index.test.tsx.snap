// Vitest Snapshot v1

exports[`Home page > should render as usual 1`] = `
<div>
  <div
    class="min-h-screen max-w-full overflow-x-hidden"
  >
    <div
      class="container mx-auto my-24"
    >
      <h1
        class="text-center text-6xl"
      >
        Nextflim
      </h1>
      <main
        role="main"
      >
        <div
          class="border-b-8 w-full text-center p-12"
          id="search-bar"
        >
          <input
            class="text-center rounded w-full md:max-w-md p-3"
            placeholder="Rechercher un film"
            value=""
          />
        </div>
      </main>
    </div>
  </div>
</div>
`;
